src/Orleans/Serialization/SerializationTestEnvironment.cs

```diff
@@ -0,0 +1,34 @@
+namespace Orleans.Serialization
+{
+    using System.Collections.Generic;
+    using System.Reflection;
+
+    using Orleans.Runtime;
+
+    public class SerializationTestEnvironment
```

Comment:
Maybe extend the comment to class documentation? Just in case someone else comes to look at this in interrim.